include/grpc++/impl/codegen/method_handler_impl.h

```diff
@@ -141,6 +141,9 @@ class ServerStreamingHandler : public MethodHandler {
     }
     ops.ServerSendStatus(param.server_context->trailing_metadata_, status);
     param.call->PerformOps(&ops);
+    if (param.server_context->has_hanging_ops_) {
```

Comment:
hanging -> pending or something else? I don't like the word hanging since it sounds like a bug.